Marlin/Marlin_main.cpp

```diff
@@ -5762,8 +5762,9 @@ inline void gcode_M17() {
     }
   }
 
-  static bool pause_print(const float& retract, const float& z_lift, const float& x_pos, const float& y_pos,
-                          const float& unload_length = 0 , int max_beep_count = 0, bool show_lcd = false) {
+  static bool pause_print(const float &retract, const float &z_lift, const float &x_pos, const float &y_pos,
+                          const float &unload_length = 0 , int8_t max_beep_count = 0, bool show_lcd = false
```

Comment:
Moving the &s to next to the variable names doesn't seem to match the code style for the rest of the file